python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2207,23 +2221,30 @@ def test_parquet_nullable_boolean(tmpdir, engine):
 )
 @pytest.mark.parametrize("index", [None, True, False])
 def test_parquet_index(pdf, index):
-    pandas_buffer = BytesIO()
-    cudf_buffer = BytesIO()
+    run_parquet_index(pdf, index)
 
-    gdf = cudf.from_pandas(pdf)
 
-    pdf.to_parquet(pandas_buffer, index=index)
-    gdf.to_parquet(cudf_buffer, index=index)
-
-    expected = pd.read_parquet(cudf_buffer)
-    actual = cudf.read_parquet(pandas_buffer)
-
-    assert_eq(expected, actual, check_index_type=True)
-
-    expected = pd.read_parquet(pandas_buffer)
-    actual = cudf.read_parquet(cudf_buffer)
-
-    assert_eq(expected, actual, check_index_type=True)
+@pytest.mark.parametrize(
+    "index",
+    [
+        pytest.param(
+            None,
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12243",
+            ),
+        ),
+        pytest.param(
+            True,
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12243",
+            ),
+        ),
+        False,
+    ],
+)
```

Comment:
One more case of mark the test to avoid `pytest.param`?

Reply:
Note that the `False` case works fine, so I need to split things out... (done).

Reply:
Thanks again... the nested parens really make this easy to miss.